src/components/scheduling/index.vue

```diff
@@ -14,7 +14,8 @@
       </ul>
     </div>
     <keep-alive>
-      <component :is="schedulingComponent" :contentId="contentId"></component>
+      <component :is="schedulingComponent" :contentId="contentId"
```

Comment:
This one is a little tricky... the `<component>` component is used for dynamic loading. Since there are actually three different components being loaded by this (website, magazine, and newsletter scheduling) the `disableDateScheduling ` option isn't applicable to magazine and newsletters. As such, I'd make a new computed property that determines which props should be sent to the component and use `v-bind` to pass them.

In other words, you'd need to add the following computed value:
```js
computed: {
  schedulingComponent() {
    const { component } = this.items.find(({ id }) => id === this.selected);
    return component;
  },
  schedulingProps() {
    const { id } = this.items.find(({ id }) => id === this.selected);
    const props = { contentId: this.contentId };
    if (id !== 'websites') return props;
    return { ...props, dateSchedulingEnabled: this.dateSchedulingEnabled };
  },
},
```

Then use `v-bind` in the template:
```vue
<component :is="schedulingComponent" v-bind="schedulingProps" />
```